### ISSUE TITLE
Minikube cannot start anymore

### ISSUE THREAD

**Issue by sebastian-305** [kubernetes/minikube#15635]
### What Happened?

Hi, I have no idea what is wrong. Here is what's happend

- Docker was installed
- I installed Minikube via winget install minikube
- I started the Minikube via minikube start
- It worked and I started the dashboard and it worked

Then everything went wrong

- I updated Docker
- I enabled Kubernetes in Docker
- I wanted to start the Minikube, but I hat the same error like in the logs
- I deleted the Minikube and startet it again
- I uninstalled it and installed it
- I restarted the system
- I uninstalled it, deleted the Image in Docker and deleted the folders I found in my home directory
- I started everything again
- Everytime the same error

I don't know if my fixing attempts made it worse but the error stayed the same. 

### Attach the log file

[logs.txt](https://github.com/kubernetes/minikube/files/10406804/logs.txt)

### Operating System

Windows

### Driver

Docker

**Comment by SneakyPanda64** [comment]
Try downgrading your docker desktop version to 4.15.0

**Comment by sebastian-305** [comment]
> Try downgrading your docker desktop version to 4.15.0

I cannot find out how to do it. As far as I can tell, they are only offering the newest version.

**Comment by SneakyPanda64** [comment]
Uninstall docker desktop then install 4.16.0 from https://docs.docker.com/desktop/release-notes/

**Comment by sebastian-305** [comment]
> Uninstall docker desktop then install 4.15.0 from https://docs.docker.com/desktop/release-notes/

Thanks, I figured that out. I wanted to find another way but I think there is none. Turned out, I kept all my images and was even still logged in after the reinstall. 

I was not aware that the update was just released. I were returning home after two weeks and startet my computer. So the problem should be, minikube has not adapted to the new update yet. Got it. Thanks for the quick support!
